docs/CompatibleSoftware.rst

```diff
@@ -19,164 +19,337 @@ application, please contact ocio-dev as well.
-the source code distribution. Of particular note are
-`src/apps/ocioconvert/
-<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ocioconvert>`__
-and `src/apps/ociodisplay/
-<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/apps/ociodisplay>`__
+Photoshop
+*********
+
+OpenColorIO display luts can be exported as ICC profiles for use in photoshop. The core idea is to create an .icc profile, with a valid description, and then to save it to the proper OS icc directory. (On OSX, ``~/Library/ColorSync/Profiles/``). Upon a Photoshop relaunch, Edit->Assign Profile, and then select your new OCIO lut.
 
-Also see the :ref:`developer-guide`
+Website : `<https://www.adobe.com/products/photoshop.html?promoid=PC1PQQ5T&mv=other>`__
 
 Python
 ******
 
-The OpenColorIO API is available for use in python. See the "pyglue" directory
-in the codebase.
+The OpenColorIO API is available for use in Python.
 
-See the developer guide for `usage examples
-<developers-usageexamples>`__ and API documentation on the PYthon
-bindings
+Website: `<https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/src/bindings/python>`__
 
-Vegas Pro
-*********
+Documentation:
+
+- `Developer Guide <https://opencolorio.org/developers/index.html>`__
+
+- `Usage Examples <https://opencolorio.org/developers/usage_examples.html>`__
+
+
+RV (Autodesk)
+*************
+
+Website : `<http://www.tweaksoftware.com>`__
+
+Supported Version:  >= 4
+
+Documentation : 
+
+- For more details, see the OpenColorIO section of the `RV User Manual <http://www.tweaksoftware.com/static/documentation/rv/current/html/rv_manual.html#OpenColorIO>`__.
+
+
+Silhouette (Boris FX)
+*********************
+
+OCIO is natively integrated in Silhouette. Full support is provided for both image import/export, as well as image display.
+
+Website : `<https://borisfx.com/products/silhouette/>`__
+
+Supported Version: >= 4.5
+
+
+Substance Designer (Adobe)
+**************************
+
+Website: `<https://www.substance3d.com/products/substance-designer/>`__
+
+Supported version: >= 2019.3
+
+Documentation:
+
+- `Color Management with OpenColorIO <https://magazine.substance3d.com/substance-designer-winter-2019-color-management-with-opencolorio/>`__
+
+- `Color Management <https://docs.substance3d.com/sddoc/color-management-188973971.html>`__
 
-`Video editing - Sony <http://www.sonycreativesoftware.com/vegaspro>`__
 
+Unreal Engine (Epic Games)
+**************************
+
+Website : `<https://unrealengine.com>`_
+
+Supported Version : >= 4.22
+
+Documentation :
+
+- `OCIO Plugin API <https://docs.unrealengine.com/en-US/API/Plugins/OpenColorIO/index.html>`_
+- `Unreal Engine 4.22 Release Notes <https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_22/index.html>`_
+
+
+Vegas Pro (Magix)
+*****************
+
+Vegas Pro 12 uses OpenColorIO, supporting workflows such as S-log footage via the ACES colorspace.
```

Comment:
Can you remove 12 from the description and instead add to this section (just to future proof it a bit):

```
Supported version: >= 12
```